front/lib/api/assistant/agent_usage.ts

```diff
@@ -0,0 +1,216 @@
+import { AgentUsageType, ModelId } from "@dust-tt/types";
+import { literal, Op } from "sequelize";
+import { v4 as uuidv4 } from "uuid";
+
+import { Mention, Message, UserMessage } from "@app/lib/models";
+import { redisClient } from "@app/lib/redis";
+
+// TTL of the ranking data in Redis.
+const rankingTTL = 60 * 60 * 24 * 30; // 30 days
+
+function _getKeys(agentConfigurationId: string) {
+  // One sorted set per agent for counting the number of times the agent has been used.
+  // score is: timestamp of each use of the agent
+  // value: random unique distinct value.
+  const agentUsedCountKey = `agent_usage_count_${agentConfigurationId}`;
```

Comment:
nit: would call it agent message count, or agent mention count
usage is very close to user, and is ambiguous, so makes the code less readable

Reply:
I know this is a big renaming since it's in a lot of places :sweat_smile: